MTM-52176 Added clarification to new asset-selector. (#189)
SoftwareAG/c8y-releasenotes: content/release-10-14-0/whatsnew-10-14-0.md

### New asset selector

> Widgets now support the new device and asset selector. The new selector allows navigating quickly through all groups and devices applicable for the currently selected widget. This feature is available to all non-datapoint related widgets.

Reviewer: ```suggestion
Widgets now support the new device and asset selector. The new selector allows navigating quickly through all groups and devices applicable for the currently selected widget. Note that this functionality is not available for widgets which are related to datapoints.
```
